src/main/java/org/folio/inventory/support/EndpointFailureHandler.java

```diff
@@ -45,6 +47,15 @@ public static void handleFailure(Throwable failure, RoutingContext context) {
     }
   }
 
+  public static <T> Function<Throwable, T> doExceptionally(
+    RoutingContext context) {
```

Comment:
Personally, I'd include the parameter on the same line, even though it is slightly over 80 characters (in practice, I typically accept between 80 and 100 characters, especially as we are likely to move towards standard that uses 100)

```suggestion
  public static <T> Function<Throwable, T> doExceptionally(RoutingContext context) {
```